signalfx-protoc/src/main/protobuf/signal_fx_protocol_buffers.proto

```diff
@@ -87,6 +87,10 @@ enum EventCategory {
      * Created by exception appenders to denote exceptional events
```

Comment:
Even though you can't actually deprecate a particular enum, i would add a comment that this is deprecated in favour of AGENT.